Lottery.java

```diff
@@ -0,0 +1,11 @@
+import java.util.Random;
+
+public class Lottery {
+    private static final int LOTTERY_MAX_COUNT = 100;
+    public Ball getRandomBall(int index, int value) {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```
This method should be without input parameters